scripts/Test-Sandbox.ps1

```diff
@@ -0,0 +1,73 @@
+﻿# Parse Arguments
+
+Param(
+  [Parameter(Mandatory, HelpMessage = "The path to the nuspec/nupkg file to test.")]
+  [String] $Nuspec
+)
+
+if (-not (Test-Path -Path $Nuspec -PathType Leaf)) {
+  throw 'The nuspec/nupkg file does not exist.'
+}
+
+# Check if Windows Sandbox is enabled
+
+if (-Not (Test-Path "$env:windir\System32\WindowsSandbox.exe")) {
+  Write-Error -Category NotInstalled -Message @'
+Windows Sandbox does not seem to be available. Check the following URL for prerequisites and further details:
+https://docs.microsoft.com/en-us/windows/security/threat-protection/windows-sandbox/windows-sandbox-overview
+
+You can run the following command in an elevated PowerShell for enabling Windows Sandbox:
+Enable-WindowsOptionalFeature -Online -FeatureName 'Containers-DisposableClientVM'
+'@ -ErrorAction Stop
+}
+
+# Initialize Temp Folder
+
+$tempFolder = Join-Path -Path $PSScriptRoot -ChildPath 'Test-Sandbox_Temp'
+
+New-Item $tempFolder -ItemType Directory -ea 0 | Out-Null
+
+Get-ChildItem $tempFolder -Recurse | Remove-Item -Force
+
+Copy-Item -Path $Nuspec -Destination $tempFolder
```

Comment:
I think I must copy the entire folder if the parameter is a nuspec file. Not needed when it's nupkg.